pkg/api/dashboard_snapshot_test.go

```diff
@@ -391,6 +390,7 @@ func setUpSnapshotTest(t *testing.T, userId int64, deleteUrl string) dashboardsn
 
 	res := &dashboardsnapshots.DashboardSnapshot{
 		ID:        1,
+		OrgID:     1,
```

Comment:
^^ the helpful change

the rest of the changes in this file just make it the same as in main (and give more clear feedback when something fails)